From 207a16bb5692c5e9be18ee04f0ba14b149808abd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Sun, 19 Apr 2026 13:51:32 +0200
Subject: [PATCH] fix

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ea780ca..1267204 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -40,7 +40,6 @@ dependencies = [
   "pathvalidate",
   "pint>=0.23",
   "plotly<6", # version 6.0.0 not fully compatible with our GUI
-  "py-spy", # for on-server profiling
   "pydantic>=2",
   "pyinstrument", # for endpoint profiling
   "pymatgen>=2025.10.7",
